isisd/isis_tlvs.c

```diff
@@ -494,36 +494,36 @@ static void format_item_ext_subtlvs(struct isis_ext_subtlvs *exts,
 						    adj->sid);
 				json_object_int_add(flags_json, "weight",
 						    adj->weight);
-				json_object_string_add(
+				json_object_boolean_add(
```

Comment:
Honestly, the JSON output of this command was not usable because the information was wrong.
For example, "show isis database json" was mixing info of all routers in a single object {}.
Even with a single router "show isis database RTR json", information was not in the right place. If multiple TLVs
were present, only the last one was displayed but sometimes with data from the previous ones.

I think it is not useful to have a deprecation cycle.